addon/-private/system/model/model.js

```diff
@@ -3,6 +3,11 @@ import { assert, deprecate } from "ember-data/-private/debug";
 import { PromiseObject } from "ember-data/-private/system/promise-proxies";
 import Errors from "ember-data/-private/system/model/errors";
 import isEnabled from 'ember-data/-private/features';
+import DebuggerInfoMixin from 'ember-data/-private/system/debug/debug-info';
```

Comment:
I guess with that, [this line in `addon/-private/system/debug.js`](https://github.com/emberjs/data/blob/9fefef6917530c7df5e148950eec8425684e5162/addon/-private/system/debug.js#L5) can be removed then: the Model is imported in [`addon/-private/system/debug/debug-adapter.js`](https://github.com/emberjs/data/blob/9fefef6917530c7df5e148950eec8425684e5162/addon/-private/system/debug/debug-adapter.js#L5)...